WireDomain/Sources/WireDomain/Repositories/Connections/Models/ConnectionInfo.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import WireDataModel
+
+public struct ConnectionInfo: Sendable {
+    public let senderID: UUID?
+    public let receiverID: UUID?
+    public let receiverQualifiedID: WireDataModel.QualifiedID?
+    public let conversationID: UUID?
+    public let qualifiedConversationID: WireDataModel.QualifiedID?
+    public let lastUpdate: Date
+    public let status: WireDataModel.ZMConnectionStatus
```

Comment:
Are you thinking to eventually have a complete domain model by not using anything from `WireDataModel`?

Reply:
yes, it'd be nice to gradually move away from WireDataModel. Although a lot was factored out, we still rely a lot on it in our local stores typically using the extension helpers from the different moc. I feel like this is also related to the discussion we have regarding a WireStorage package (and the spike ticket), having a fully isolated storage layer that doesn't depend anymore on WireDataModel